python/pyxet/pyxet/file_system.py

```diff
@@ -578,6 +578,35 @@ def mv(self, path1, path2, *args, **kwargs):
     def move(self, path1, path2, *args, **kwargs):
         return self.mv(path1, path2, *args, **kwargs)
 
+    def add_deduplication_hints(self, path_urls, min_dedup_byte_threshhold = None):
```

Comment:
I'm not sure if this threshold should be exposed to user to set, because they have no idea of shard structures and this is anyway a ballpark number.

Reply:
This parameter doesn't require shards to understand.   But I did forget to document it.